server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_11_alter_regulation_riskdetect_rule.sql

```diff
@@ -0,0 +1,4 @@
+--
+-- Alter column `value_json` size from varchar(2048) to varchar(10240) in `regulation_riskdetect_rule` table
+--
+alter table `regulation_riskdetect_rule` modify column `value_json` mediumtext default null COMMENT 'json of rule value';
```

Comment:
it is not compatible with OB227x ?
may use CHANGE COLUMN + add COLUMN replacement,

refer V_3_2_2_1__add_and_rename_task_info_column.sql for details

Reply:
got it, job finished.